force-app/main/default/classes/TestContactController.cls

```diff
@@ -117,6 +117,34 @@ public with sharing class TestContactController {
             }
             Test.stopTest();
         }
+    }
+
+    @isTest
+    static void updateContact() {
+        TestContactController.createContacts(1);
+        Contact contact = [SELECT Id, FirstName, LastName FROM Contact LIMIT 1];
+        String firstName = 'Changed ' + contact.FirstName;
+        String lastName = 'Changed ' + contact.LastName;
+
+        Test.startTest();
+        ContactController.updateContact(contact.Id, firstName, lastName);
+        Contact updatedContact = [
+            SELECT Id, FirstName, LastName
+            FROM Contact
+            WHERE Id = :contact.Id
+        ];
+        Test.stopTest();
 
+        System.assertEquals(
```

Comment:
Use new `Assert` class for assertions.

Reply:
Used Assert class in all the test classes.